_posts/en/newsletters/2021-08-18-newsletter.md

```diff
@@ -0,0 +1,140 @@
+    by creating UTXOs that the users will have no financial incentive to
+    ever spend.  Parts of the discussion also [described][riard dust]
+    the [impact][towns dust] of both the dust limit and uneconomical
+    outputs on parts of LN.
+
+    As of this writing, it did not appear any agreement was likely to be
+    reached.  At least for the short term, we expect the dust limit to
+    remain.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Preparing for taproot #9: signature adaptors
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/08-signature-adaptors.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  and enhanced support for hardware wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #22642][] updates Bitcoin Core's release process for
+  upcoming version 22.0 to concatenate the GPG signatures of everyone
+  who [reproducibly built][topic reproducible builds] the binaries into a single file which
+  can be batch verified ([example][gpg batch]).  Signatures from
+  determinsitic builders have been available for years, but this should
+  make them more accessible and also reduce the existing dependency
+  on the project's lead maintainer signing the release binaries.
+
+- [Bitcoin Core #21800][] implements ancestor and descendant limits for
+  mempool package acceptance. Bitcoin Core limits the number of
+  related transactions in its mempool as a protection against DoS
+  attacks and so that block construction is tractable for miners. By default,
+  those [limits][bitcoin core mempool limits] ensure that no transaction in
+  the mempool can have  more than 25 ancestors or descendants, and that no
+  transaction can have ancestors or descendants that have a combined weight of
+  101kVb.
```

Comment:
The limits are inclusive of the tx itself. Also, it's kind of hard to word this in a clear way, but I hope to articulate that there are a total of 4 limits:

```suggestion
- [Bitcoin Core #21800][] implements ancestor and descendant limits for
  mempool package acceptance. Bitcoin Core limits the number of
  related transactions in its mempool as a protection against DoS
  attacks and so that block construction is tractable for miners. By default,
  those [limits][bitcoin core mempool limits] ensure that no transaction
  in the mempool, combined with its mempool ancestors, can exceed 25
  transactions or 101KvB in weight. The same rules apply to the transaction
  combined with its mempool descendants.
```